collector/Makefile

```diff
@@ -17,19 +17,21 @@ COLLECTOR_BUILD_DEPS := $(HDRS) $(SRCS) $(shell find falcosecurity-libs -name '*
 
 .SUFFIXES:
 
-cmake-build/collector: $(COLLECTOR_BUILD_DEPS)
-	docker rm -fv build_collector || true
-	docker run --rm --platform ${PLATFORM} --name build_collector \
-		-v "$(BASE_PATH):$(SRC_MOUNT_DIR)" \
-		-e NPROCS=$(NPROCS) \
-		-e SRC_ROOT_DIR="$(SRC_MOUNT_DIR)" \
-		-e CMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE) \
-		-e USE_VALGRIND=$(USE_VALGRIND) \
-		-e TRACE_SINSP_EVENTS=$(TRACE_SINSP_EVENTS) \
-		-e ADDRESS_SANITIZER=$(ADDRESS_SANITIZER) \
-		-e COLLECTOR_APPEND_CID=$(COLLECTOR_APPEND_CID) \
-		-e DISABLE_PROFILING="true" \
-		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG) "$(SRC_MOUNT_DIR)/builder/build/build-collector.sh"
+cmake-configure/collector:
+	docker exec $(COLLECTOR_BUILDER_NAME) \
+		cmake -S $(BASE_PATH) -B $(CMAKE_DIR) \
+			-DCMAKE_BUILD_TYPE=$(CMAKE_BUILD_TYPE) \
+			-DDISABLE_PROFILING=ON \
+			-DUSE_VALGRIND=$(USE_VALGRIND) \
+			-DADDRESS_SANITIZER=$(ADDRESS_SANITIZER) \
```

Comment:
Originally there were some default values specified for this variables in the build-collector script. Are they defined somewhere else now?

Reply:
Always have been: https://github.com/stackrox/collector/blob/6c763b76bcee5c66ae7bc92b475d9171378f33c3/Makefile-constants.mk#L14-L19